pwa/src/components/soaps/soapForm.js

```diff
@@ -107,7 +107,7 @@ export default function SoapsForm({id}) {
     // This removes empty values from the body
     body = removeEmptyObjectValues(body);
 
-    if (!checkValues([body.name, body.type, ])) {
+    if (!checkValues([body.name, body.type,])) {
```

Comment:
Deze trailing comma is niet nodig, dus:

... body.type, => ...body.type

Reply:
comma is het verwijderd